Applications/Analyze/test/testAnalyzeTool.cpp

```diff
@@ -140,12 +146,15 @@ void testTugOfWar(const string& dataFileName, const double& defaultAct) {
     size_t nstates = statesTraj.getSize();
 
     // muscle active, passive, total muscle and tendon force quantities
-    double af, pf, mf, tf;
-    af= pf = mf = tf = SimTK::NaN;
+    double af, pf, mf, tf, fl, fv;
+    af= pf = mf = tf = fl = fv = SimTK::NaN;
 
     // Tolerance for muscle equilibrium solution 
     const double equilTol = muscle.getMaxIsometricForce()*SimTK::SqrtEps;
 
+    // The maximum acceptable change in force between two contiguous states
+    const double maxDelta = muscle.getMaxIsometricForce() / 2;
```

Comment:
Good point. I simply used a value that was sufficient to detect the skyscraper bug. But now knowing what are the typical transitions I will tighten up.